tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -69,7 +69,9 @@ Docker logs backward compatibility
     Should Contain  ${output}  container ${id1} does not support '--timestamps'
```

Comment:
Docker logs backward compatibility test doesn't support using Harbor- test were failing and Mike suggested reverting it back